analyzer/consensus/consensus.go

```diff
@@ -36,6 +38,12 @@ import (
 
 const (
 	consensusAnalyzerName = "consensus"
+
+	// The query takes ~20 second to run on mainnet at the time of implementing this.
+	accountListViewRefreshInterval = 2 * time.Minute
+	accountsListViewRefreshTimeout = 2 * time.Minute
+	accountListViewRefreshQuery    = `REFRESH MATERIALIZED VIEW CONCURRENTLY views.accounts_list`
+	accountListOpName              = "consensus_account_list_refresh"
```

Comment:
Looks great, that was a quick refactor 😄 

> hurdle was ensuring the analyzer was run once after the (slow) consensus block-based analyzer 

Ah so we actually ran into this issue before for other item based analyzers, so we made the e2e_regression tests  run analyzers in 2 separate stages denoted in `e2e_config_1.yml` and `e2e_config_2.yml`. The first stage has all the block analyzers, and the second stage runs the item-based analyzers that do subsequent processing of the data ingested by the first stage. The `accounts_list` analyzer probably belongs in the second stage, which should simplify the `computed_height`/queueLength.

Reply:
Oh yeah that makes sense. I'll move it into the second stage. Will probably keep the the computed_height - since it might end up being useful for debugging purposes.